lib/node_modules/@stdlib/math/base/special/gammasgn/test/test.js

```diff
@@ -0,0 +1,165 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2022 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var isPositiveZero = require( '@stdlib/math/base/assert/is-positive-zero' );
+var EPS = require( '@stdlib/constants/float64/eps' );
+var abs = require( '@stdlib/math/base/special/abs' );
+var gammasgn = require( './../lib' );
+
+
+// FIXTURES //
+
+var random = require( './fixtures/python/random.json' );
+
+var mediumNegative = require( './fixtures/python/medium_negative.json' );
+var mediumPositive = require( './fixtures/python/medium_positive.json' );
+var smallPositive = require( './fixtures/python/small_positive.json' );
+var smallNegative = require( './fixtures/python/small_negative.json' );
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.equal( typeof gammasgn, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function computes the sign of `gamma` evaluated in `x`', function test( t ) {
+	var expected;
+	var delta;
+	var tol;
+	var x;
+	var v;
+	var i;
+
+	x = random.x;
+	expected = random.expected;
+
+	for ( i = 0; i < x.length; i++ ) {
+		v = gammasgn( x[ i ] );
+		delta = abs( v - expected[ i ] );
```

Comment:
None of this should be needed. The value comparison should be exact, as the function can only return `1.0`, `-1.0`, or `0.0`.